mirar/paths.py

```diff
@@ -17,6 +19,9 @@
 
 doc_dir = base_code_dir.joinpath("docs/")
 
+# Load environment variables for .env file
+dotenv.load_dotenv()
```

Comment:
What happens if there is no .env file? Does it silently pass, or does it fail?

I would prefer something with the path:

```
dotenv_paths = [code_dir.glob('*.env')]
assert len(dotenv_paths) < 2, "Multiple dotenv paths found" # or similar check
if len(dotenv_paths) > 0:
    path = dotenv_paths[0]
    if dotenv_path.exists():
          logger.info(f"Found .env file {dot_env}, loading from here.")
          load_dotenv(path)
```

Reply:
(That's roughly pseudo-code, I know there are a couple of typos but you get the gist I think)

Reply:
It silently passes if there's no .env file.

I'm not sure if there's a need to search for different "*.env" since the actual file itself should always just be called ".env". As in, it should be a "hidden" file similar to other types of configuration files.

Reply:
OK, makes sense!